design-documents/graph-ql/coverage/negotiableQuotes.graphqls

```diff
@@ -1,205 +1,332 @@
-    discounts: [Discount] @doc(description:"An array of discounts to be applied to the quote item")
-    total_item_discount: Money @doc(description:"The total of all discounts applied to the item")
+input NegotiableQuoteItemQuantityInput {
+    quote_item_id: ID!
+    quantity: Float!
 }
 
-type Discount @doc(description:"Defines an individual discount. A discount can be applied to the quote as a whole or to an item.") {
-    amount: Money! @doc(description:"The amount of the discount")
-    label: String! @doc(description:"A description of the discount")
+input RequestNegotiableQuoteInput {
+    cart_id: ID!
+    quote_name: String!
+    comment: NegotiableQuoteCommentInput!
+    # files (attachments) to be added at a later date when file upload design has been finalized
 }
 
-type DefaultNegotiableQuoteItem implements NegotiableQuoteItemInterface
-@doc(description: "Negotiable Quote Item Implementation for Simple and Virtual Products") {
-    id: ID! @doc(description: "Unique Identifier of Negotiable Quote Item")
-    product: ProductInterface!
-    qty: Float! @doc(description:  "Quantity added")
-    customizable_options: [SelectedCustomizableOption] @doc(description: "custom Option selected")
+input NegotiableQuoteCommentInput {
+    comment: String!
+    # files (attachments) to be added at a later date when file upload design has been finalized
 }
 
-type DownloadableRequisitionListItem implements NegotiableQuoteItemInterface
-@doc(description: "Negotiable Quote Item Implementation that for Downloadable Products") {
-    id: ID! @doc(description: "Unique Identifier of Negotiable Quote Item")
-    product: ProductInterface!
-    qty: Float! @doc(description: "Quantity added")
-    customizable_options: [SelectedCustomizableOption] @doc(description:  "custom Option selected")
-    links: [DownloadableProductLinks] @doc(description: "DownloadableProductLinks defines characteristics of a downloadable product")
-    samples: [DownloadableProductSamples] @doc(description:  "DownloadableProductSamples defines characteristics of a downloadable product")
+input AddNegotiableQuoteCommentInput {
+    quote_id: ID!
+    comment: NegotiableQuoteCommentInput!
 }
 
-type BundleRequisitionListItem implements NegotiableQuoteItemInterface
-@doc(description: "Negotiable Quote Item Implementation that for Bundle Products") {
-    id: ID! @doc(description: "Unique Identifier of Negotiable Quote Item")
-    product: ProductInterface!
-    qty: Float! @doc(description: "Quantity added")
-    customizable_options: [SelectedCustomizableOption] @doc(description: "custom Option selected")
-    bundle_options: [SelectedBundleOption]! @doc(description: "selected bundle options")
+type NegotiableQuoteComment {
+    id: ID!
+    created_at: String!
+    author: NegotiableQuoteUser!
+    creator_type: NegotiableQuoteCommentCreatorType!
+    value: String! @doc(description: "A simple (non-html) comment submitted by a seller or buyer")
```

Comment:
How about
```suggestion
    text: String! @doc(description: "A simple (non-html) comment submitted by a seller or buyer")
```

Reply:
I went with `value` only because it seems to be the most common name for this type of `String` field in our schema.

Just confirm if you think `text` provides more clarity, and I'll gladly change :)

Reply:
In the past we used `text` for such cases, not critical though.

Reply:
I changed it over to `text` 👍